prog/dftb+/api/mm/mmapi.F90

```diff
@@ -467,6 +473,64 @@ subroutine convertAtomTypesToSpecies(typeNumbers, species, speciesNames, typeNam
 
   end subroutine convertAtomTypesToSpecies
 
+  !TODO(Alex) Consider whether this should be a member function or free function
```

Comment:
Consider whether this should be a member function or free function

Reply:
It should remain a member function.